spec/requests/citizens/student_finance/annual_amounts_spec.rb

```diff
@@ -45,6 +43,21 @@
         expect(irregular_income.frequency).to eq 'annual'
         expect(irregular_income.income_type).to eq 'student_loan'
       end
+
+      describe 'update record' do
+        before { patch citizens_student_finances_annual_amount_path, params: params }
+
+        context 'update amount' do
+          let(:amount) { 5000 }
+          it 'updates the same record without creating a new one' do
+            expect { patch citizens_student_finances_annual_amount_path, params: params }.to change { IrregularIncome.count }.by(0)
+            irregular_income = legal_aid_application.irregular_incomes.first
+            expect(irregular_income.amount).to eq 5000
+            expect(irregular_income.frequency).to eq 'annual'
```

Comment:
we could probably remove the lines that test 'annual' and 'student_loan' as we already test these two above.

Reply:
👍